src/ucp/core/ucp_request.h

```diff
@@ -117,17 +117,28 @@ struct ucp_request {
             ucp_send_callback_t   cb;       /* Completion callback */
 
             union {
-
                 ucp_wireup_msg_t  wireup;
 
-                /* Tagged send */
                 struct {
-                    ucp_tag_t        tag;
-                    uint64_t         message_id;  /* message ID used in AM */
-                    ucp_lane_index_t am_bw_index; /* AM BW lane index */
-                    uintptr_t        rreq_ptr;    /* receive request ptr on the
-                                                     recv side (used in AM rndv) */
-                } tag;
+                    ucp_lane_index_t am_bw_index;     /* AM BW lane index */
```

Comment:
please remove this refactoring.
better do in other PR and name common structure somehow